client/src/pages/Profile/Profile.tsx

```diff
@@ -0,0 +1,74 @@
+import TextField from '@mui/material/TextField';
+import AdapterDateFns from '@mui/lab/AdapterDateFns';
+import LocalizationProvider from '@mui/lab/LocalizationProvider';
+import DateTimePicker from '@mui/lab/DateTimePicker';
+import { mockProfile } from '../../mocks/mockProfile';
+export default function Profile(): JSX.Element {
+  const classes = useStyles();
+  const [dropInValue, setDropInValue] = useState<Date | null>(new Date());
+  const [dropOffValue, setDropOffValue] = useState<Date | null>(new Date());
+  const { firstName, lastName, introduction, address, description, galleryPics, profilePic, coverPic } = mockProfile;
+  return (
+    <Grid container>
+      <Paper className={classes.profileContainer}>
+        <img className={classes.coverImage} src={coverPic} alt="Cover Photo" />
+        <Grid container className={classes.basicInfoContainer} direction="column" alignItems="center">
+          <img className={classes.profilePic} src={profilePic} alt="Profile Pic" />
+          <Typography variant="h4">
+            {firstName} {lastName}
+          </Typography>
+          <Typography variant="subtitle1">{introduction}</Typography>
+
+          <Typography color="primary" variant="subtitle2">
+            <LocationOnIcon fontSize="small" className={classes.locationIcon} /> {address}
+          </Typography>
+        </Grid>
+        <Grid container className={classes.aboutContainer}>
+          <Typography variant="h5">About me</Typography>
+          <Typography variant="body1" className={classes.description}>
+            {description}
+          </Typography>
+          <Grid container className={classes.galleryContainer}>
+            {galleryPics?.map((url) => (
+              <img key={url} src={url} className={classes.galleryPic} alt="Galley Pics" />
+            ))}
+          </Grid>
+        </Grid>
+      </Paper>
+      <Paper component="form" className={classes.bookingContainer}>
+        <LocalizationProvider dateAdapter={AdapterDateFns}>
+          <Grid container className={classes.requestContainer}>
+            <Typography variant="h5">$14/hr</Typography>
+            <Grid container direction="column" className={classes.dateContainer}>
+              <DateTimePicker
+                renderInput={(props) => <TextField {...props} />}
+                label="Drop In"
+                value={dropInValue}
+                onChange={(newValue) => {
+                  setDropInValue(newValue);
+                }}
+              />
+            </Grid>
+            <Grid container direction="column" className={classes.dateContainer}>
+              <DateTimePicker
+                renderInput={(props) => <TextField {...props} />}
+                label="Drop Off"
+                value={dropOffValue}
+                onChange={(newValue) => {
+                  setDropOffValue(newValue);
+                }}
+              />
+            </Grid>
+            <Button type="submit" color="primary">
```

Comment:
Please add a variant to this button to match the button to the mockup - `variant="contained"`

Reply:
@rajivtitus  Variant button added how ever the color primary is now giving blue color instead of our theme, right now i have kept it color="error" but i would rather use theme. so any ideas??

Reply:
@mayanksuchde This article below has relevant information which should solve your problem :) 

Ref: https://stackoverflow.com/questions/64271903/react-button-does-not-change-its-color-after-material-ui-theme-changed

Reply:
Hey @rajivtitus , So the reason it was not taking the theme color is because this button was imported from @mui/material and not @material-ui/core plus the createMuiTheme is from @material-ui/core which is also deprecated , So I created a new theme in Profile.tsx from @mui/material  and provide it to the button,

Reply:
@mayanksuchde Ahhh I see! Good catch 👍🏽